src/app/task/task.component.html

```diff
@@ -1,25 +1,28 @@
 <h2>Task To Do: </h2>
 
 <div>
-
     <form [formGroup]="taskForm" (ngSubmit)="addTask()">
-        <label> Name:
-      <input type="text" formControlName="name">
-    </label>
+        <label for="form1" class=''> Name:
+            <input mdbInput type="text" id="form1" class="form-control" formControlName="name">
```

Comment:
You have used from1 id twice. Id's on elements should be unique throughout whole template.